client/ts/src/client.ts

```diff
@@ -475,34 +484,53 @@ export class ManifestClient {
  * Same as the autogenerated WrapperPlaceOrderParams except price here is a number.
  */
 export type WrapperPlaceOrderParamsExternal = {
-  /** Number of base atoms in the order. */
-  baseAtoms: bignum;
-  /** Price as float in atoms of quote per atoms of base. */
-  price: number;
+  /** Number of base tokens in the order. */
+  numBaseTokens: number;
+  /** Price as float in quote tokens per base tokens. */
+  tokenPrice: number;
   /** Boolean for whether this order is on the bid side. */
   isBid: boolean;
   /** Last slot before this order is invalid and will be removed. */
   lastValidSlot: number;
   /** Type of order (Limit, PostOnly, ...). */
   orderType: OrderType;
   /** Used in fill or kill orders. Set to zero otherwise. */
-  minOutAtoms?: bignum;
+  minOutTokens?: number;
   /** Client order id used for cancelling orders. Does not need to be unique. */
-  clientOrderId: bignum;
+  clientOrderId: number;
```

Comment:
client order id is a u64 onchain. Any reason not to use a big num? If we only ever will use smaller then alright to do number.

Reply:
*Shrug* just confusing to users probably. Going for ease of use but can add back if you think it's better to have the functionality there

Reply:
main concern is that some people might use milliseconds since epoch for client order id. otherwise, just make sure we use integers only

Reply:
changed back to bignum in https://github.com/CKS-Systems/manifest/pull/50